src/index.css

```diff
@@ -1,27 +1,52 @@
-body {
+* {
   margin: 0;
-  font-family:
-    -apple-system,
-    BlinkMacSystemFont,
-    'Segoe UI',
-    'Roboto',
-    'Oxygen',
-    'Ubuntu',
-    'Cantarell',
-    'Fira Sans',
-    'Droid Sans',
-    'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
 }
 
-code {
-  font-family:
-    source-code-pro,
-    Menlo,
-    Monaco,
-    Consolas,
-    'Courier New',
-    monospace;
+input {
+  width: 399px;
+  height: 70px;
+  background-color: #47484f;
+  border: none;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  top: 50%;
+  left: 50%;
```

Comment:
- [ ] It would be better if your  Calculator.js component matches the following given design in colors and please display `0` on the calculator's screen 👇 
<img width="558" alt="calculator" src="https://user-images.githubusercontent.com/67087664/170057725-91beeb09-ac83-492f-9a75-f528437a69bc.png">